make/releasing.mk

```diff
@@ -11,6 +11,10 @@ build:
 akash:
 	$(GO) build $(BUILD_FLAGS) ./cmd/akash
 
+.PHONY: akash_docgen
```

Comment:
`cmddoc` ?

Reply:
Are you suggesting that as the name of the executable we create?